conda/environments/cudf_dev_cuda9.2.yml

```
@@ -1,10 +1,8 @@
 name: cudf_dev
 channels:
-- nvidia
 - rapidsai
```

I think this should be 
```suggestion
- rapidsai/label/cuda9.2
- nvidia/label/cuda9.2
```

---

You also need a changelog...